### ISSUE TITLE
[MU4 Issue] Muse Hub fails to launch

### ISSUE THREAD

**Issue by SteveBlower** [musescore/MuseScore#15173]
**Describe the bug**
After new installation of release version of Muse Hub it fails to launch.

**To Reproduce**

On Windows 10

Uninstall development versions of Muse Hub and MuseScore V4
Download release version of Muse Hub
Click on downloaded Muse Hub.exe
respond to "Allow Muse Hub.exe to make changes"
See message that Muse Hub will launch when installation is complete

... Nothing

I can see Muse.Service.exe in task manager but not Muse Hub. I can see Muse Hub in the start menu but clicking on it has no effect, not even a brief appearance in Task Manager.

**Expected behavior**
A clear and concise description of what you expected to happen

Muse Hub launches after installation or on clicking Muse Hub icon in start menu.

**Screenshots**

**Platform information**
Windows 10

**Additional context**

See also https://musescore.org/en/node/337823#comment-1153175

**Comment by SteveBlower** [comment]
Here it is.  

I have tried re-installing a few times but no improvement.  I will try again after doing a reboot later today - currently  busy with the PC on WORK stuff.

[Installer.log](https://github.com/musescore/MuseScore/files/10230225/Installer.log)

**Comment by DominicMaas** [comment]
No problem, when you have the time, can you try installing the Windows App SDK Dependency (preferably as admin)?

https://aka.ms/windowsappsdk/1.2/1.2.221116.1/windowsappruntimeinstall-x64.exe

It should have been installed (and the logs say it had), but I know this has worked for people in the past (it may have only been partially installed)

**Comment by SteveBlower** [comment]
Installed Windows App SDK Dependency as admin
Rebooted
Muse Hub runs but gives "Error Muse Hub helper service is not running"
Hitting Connect button just leaves it greyed out.
I see Muse Hub in task manager
I don't see Muse.Service.exe in task manager as I did with previous Dev versions.

**Comment by DominicMaas** [comment]
Can you go to "Services" and ensure the Muse Hub service is running (right click -> start).

And hmmmmm. I think the Microsoft Store version of Windows App SDK might be breaking stuff on Windows 10... (maybe https://github.com/microsoft/WindowsAppSDK/issues/3137 , but that fix is only in Jan....)

**Comment by DominicMaas** [comment]
Also, if you could open "Event Viewer" and go to "Windows Logs" -> "Application". Are there any Muse Hub crashes logged there recently?

**Comment by SteveBlower** [comment]
Windows wanted to restart for an update after installing Windows App SDK.  After rebooting still not luck with Muse Hub.  Then I looked at the services in task manager (I actually use Process Hacker) and found Muse.Service.exe was there but Stopped.  I started it and now Muse Hub seems to be working OK.

**Comment by Gravifer** [comment]
I came across a similar issue, however the service indeed show up in taskmgr and mmc. The process however shows 0% CPU usage, and about 51MB unchanging RAM usage. I suspect it is deadlocked, although the mmc says that the service is running properly. The problem persists after a few reinstall tries. Sadly I installed the bare version of MuseScore 4 before searching for this problem, and I only have the failed installation log confirming I indeed have the App SDK before all of this. I removed MuseScore and tried to reproduce the scene, but the app works this time, both the taskbar icon and the main interface are functioning. The log this time shows nothing surprising. 

I have tested the beta version, no problem occured during installlation, and I removed the beta soon after that. 

Hope my diatribe helps with your triage.
